GAME/TYPES.H

```diff
@@ -14,7 +14,15 @@
 #endif
 
 #ifndef NULL
-#define NULL 0
+#define NULL ((void*)0)
+#endif
+
+#ifndef FALSE
```

Comment:
Definition of TRUE/FALSE is good, just be careful as it's possible some variables may not be bools.